ramalama/oci.py

```diff
@@ -53,16 +55,47 @@ def _target_decompose(self, model):
         reference_dir = reference.replace(":", "/")
         return registry, reference, reference_dir
 
-    def push(self, source, args):
-        target = args.TARGET.strip("oci://")
-        tregistry, _, treference_dir = self._target_decompose(target)
+    def _build(self, source, target, args):
+        print(f"Building {target}...")
+        src = os.path.realpath(source)
+        model_name = os.path.basename(source)
+        contextdir = os.path.dirname(src)
+        model = os.path.basename(src)
+        containerfile = tempfile.NamedTemporaryFile(prefix='RamaLama_Containerfile_', delete=False)
+        # Open the file for writing.
+        with open(containerfile.name, 'w') as c:
+            c.write(
+                f"""\
+FROM {args.image} as builder
+RUN mkdir -p /run/model; cd /run/model; ln -s {model_name} model.file
+
+FROM scratch
+COPY --from=builder /run/model /
+COPY {model} /{model_name}
+"""
+            )
+        run_cmd([self.conman, "build", "-t", target, "-f", containerfile.name, contextdir], stdout=None)
```

Comment:
wouldn't this require `conman` to be available a system tool/dependency? 🤔

Reply:
Being able to create and push to a OCI Registry requires an OCI Container Engine.